src/main/java/com/netgrif/application/engine/workflow/web/AbstractTaskController.java

```diff
@@ -190,17 +190,35 @@ public CountResponse count(SingleElasticTaskSearchRequestAsList query, MergeFilt
 
 
     public EntityModel<EventOutcomeWithMessage> getData(String taskId, Locale locale) {
+        try {
         GetDataGroupsEventOutcome outcome = dataService.getDataGroups(taskId, locale);
         return EventOutcomeWithMessageResource.successMessage("Get data groups successful",
                 LocalisedEventOutcomeFactory.from(outcome,locale));
+        } catch (Exception e) {
```

Comment:
do we really want to catch everything? How was the error-handling done so far?

Reply:
i unified error handling implementation with assign, finish, cancel and delegate